### DISCUSSION TITLE
What is medialib ?

### DISCUSSION THREAD

**Discussion by Npa972** [DeaDBeeF-Player/deadbeef#3016] [Q&A] [answered]
In the latest build, there is a new Medialib plugin.
Will it automatically update the library like Rhythmbox for example ?
Thanks

**Comment by Npa972** [comment, marked as answer]
OK I found it. Now I have to look at the configuration possibilities...

**Comment by TheOddler** [comment]
Where is this plugin? Does it no longer exist or am I missing something?

**Reply by TheOddler** [reply]
I found in the source code that there's supposed to be a medialib plugin, but weirdly it's not in my list of included plugins:
![image](https://github.com/user-attachments/assets/5990703b-41e9-479e-bbb9-c562f87bf3f1)

**Reply by TheOddler** [reply]
Looks like it needs to be enabled somehow: https://github.com/DeaDBeeF-Player/deadbeef/blob/master/plugins/medialib/Makefile.am#L1

Though if I look at the code, it is enabled by default: https://github.com/DeaDBeeF-Player/deadbeef/blob/afff705b2f4b05813ebcc54e343e7e20e79a6082/configure.ac#L128

So not sure why it isn't there in my distro's package (I'm using NixOS). I'll check with the maintainer there, or see if I can figure it out myself.

**Reply by Oleksiy-Yakovenko** [reply]
It's coming in next release 1.10, you can try release candidate build if you want

**Reply by TheOddler** [reply]
Awesome! That's why my attempts of enabling it in 1.9.6 didn't work, the `--enable-medialib` flag doesn't exist there yet! I'll have a look at the release candidate. Thank you!
